docs/aws-professional/services.md

```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
 
 ## Marketplace
 
-| Area        | AWS service     | Azure service                                                 | Description                                                                                                                                   |
-|-------------|-----------------|---------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
-| Marketplace | AWS Marketplace | [Azure Marketplace](https://azure.microsoft.com/marketplace/) | Easy-to-deploy and automatically configured third-party applications, including single virtual machine or multiple virtual machine solutions. |
+| Area        | AWS service     | Azure service                                                 | Description                                                                                                                                               |
+|-------------|-----------------|---------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Marketplace | AWS Marketplace | [Azure Marketplace](https://azure.microsoft.com/marketplace/) | Easy-to-deploy and automatically configure third- AWS Marketplace Marketplace party applications, including single or multiple virtual machine solutions. |
```

Comment:
Something garbled here? ("third- AWS Marketplace Marketplace party applications")

Reply:
Yes. the corrected text is "Easy-to-deploy and automatically configure third-party applications, including single or multiple"
virtual machine solutions.